reconcile/openshift_users.py

```diff
@@ -14,21 +15,38 @@
 QONTRACT_INTEGRATION = "openshift-users"
 
 
-def get_cluster_users(cluster, oc_map):
+def get_cluster_users(cluster, oc_map, clusters):
     oc = oc_map.get(cluster)
     if not oc:
         logging.log(level=oc.log_level, msg=oc.message)
         return []
-    users = [
-        u["metadata"]["name"]
-        for u in oc.get_users()
-        if u.get("identities", None) is not None
-        and len(u["identities"]) == 1
-        and u["identities"][0].startswith("github")
-        and not u["metadata"].get("labels", {}).get("admin", "")
-    ]
+    users: list[str] = []
+
+    cluster_info = next((cl for cl in clusters if cl["name"] == cluster), None)
```

Comment:
if cluster_info can't be None (and from the rest of the code my understanding is as well that it can't be None), the following change would spare you from the `None` check (i assume it is mostly here because of mypy)
```suggestion
    cluster_info = next((cl for cl in clusters if cl["name"] == cluster))
```

Reply:
That does the trick. Thx